### DISCUSSION TITLE
How can I use test step when implementing automation tests using Vitest?

### DISCUSSION THREAD

**Discussion by dmanukyan** [vitest-dev/vitest#5373] [General]
Does Vitest have a feature like a test.step(test body)

**Comment by sheremet-va** [comment]
Vitest doesn't have a `test.step` feature

**Reply by dmanukyan** [reply]
Are there any features to separate the test steps?

**Reply by sheremet-va** [reply]
No, there is no concept of test steps in Vitest

**Reply by dmanukyan** [reply]
Okay, thank you!
